### ISSUE TITLE
angular 1.6.1, angular-material 1.1.3 -- UNMET PEER DEPENDENCIES

### ISSUE THREAD

**Issue by simonemazzoni** [angular/material#10353]
**Actual Behavior**:
 -  `What is the issue? npm install error "UNMET PEER DEPENDENCY" for packages angular, angular-animate and angular-aria with versions 1.6`
 -  `What is the expected behavior?`

**Angular Versions**: *
 -  `Angular Version: 1.6.1`
 -  `Angular Material Version: 1.1.3`

**Additional Information**:
 -  `Browser Type: Chrome`
 -  `Browser Version: 56.0.2924.76 (Official Build) (64-bit)`
 -  `OS: macOS sierra`

-----
The angular-material 1.1.3 package.json defines this peerDependencies
`"peerDependencies": {
    "angular": ">=1.3 <1.6",
    "angular-animate": ">=1.3 <1.6",
    "angular-aria": ">=1.3 <1.6"
  },`
even if your milestone description says that version [1.1.3](https://github.com/angular/material/milestone/37) should fix angular 1.6.x compatibility.

I guess this solves the issue
`"peerDependencies": {
    "angular": ">=1.3 <=1.6",
    "angular-animate": ">=1.3 <=1.6",
    "angular-aria": ">=1.3 <=1.6"
  },`

but now I'm wandering, Is angular-material 1.1.3 actually compatible with angular 1.6.x?

Simone

**Comment by vbraun** [comment]
There are some open tickets with angular 1.6.x regressions, presumably thats the reason: https://github.com/angular/material/issues?q=1.6.0+label%3A%22needs%3A+Angular+1.6+support%22

**Comment by simonemazzoni** [comment]
@vbraun I see, but they are pretty old, so I guessed it was an error.

Will this be fixed in these days?

Thanks

**Comment by surfjedi** [comment]
Its just got to do with the package.json declarations.  I have a PR on it here:
https://github.com/angular/material/pull/10292

**Comment by C-F-4** [comment]
Getting UMET PEER DEPENDENCIES for angular-aria and angular-material too!!

**Comment by surfjedi** [comment]
@C-F-4 I hopefully covered that in my PR. But this is my first PR to material, I think they need to review it.

**Comment by graingert** [comment]
duplicate of #10111

**Comment by fnc314** [comment]
Any resolution on this?  Looking to migrate a large app to 1.6.x and would like to keep Material 1.1.3.  Thanks for all the hard work Angular Team, I wish I was 1/2 as good as you all.

**Comment by graingert** [comment]
@kara can you close this as duplicate?

**Comment by simonemazzoni** [comment]
@graingert I know it looks like a duplicate, but the duplicate post is two months old, and it hasn't been solved yet. 
Is this project still maintained? I see the number of issues and PR increase every day.

**Comment by graingert** [comment]
@simonemazzoni @kara made a commit 18 days ago. Seems alive to me.

**Comment by simonemazzoni** [comment]
@graingert k, I was just asking because the dates of the milestones looks quite strange to me.
Do I have to ignore that dates?

Anyway I'll wait for the fix then! Keep up the good work! Angular-Material is awesome.

**Comment by graingert** [comment]
@ThomasBurleson can you re-open this PR until 1.1.4 or greater is released?

**Comment by ThomasBurleson** [comment]
Waiting on release of 1.1.4.

**Comment by setry14** [comment]
For what I have seen, angularjs 1.6 .3 components work just fine with  angular material 1.1.3. it's just that this unmet peer dependency warning keeps coming on npm install. This warning makes no sense to me if the there are no functional issues. Hope 1.1.4 fixes this warning soon​.

**Comment by PedroF20** [comment]
Also waiting on the release of 1.1.4

**Comment by surfjedi** [comment]
All this needed was a change to deps in the package.json. I already fixed it and there is pr on it.

See the link i put above

**Comment by simonemazzoni** [comment]
@surfjedi I see, but I guess all are waiting for the official update to keep all "clean".
The problem is that the milestone for version `1.1.4` continue to be delayed.

**Comment by calebtote** [comment]
I've tried the "hack" for `package.json`, but it seems to be auto-reverting my changes from `<=1.6` to `<1.6` when running the install string:

`$ npm install angular angular-animate angular-aria angular-material angular-messages mdi`

**Comment by bmetcalfe1** [comment]
Any update on the update? :) I'm having the same issue

**Comment by EladBezalel** [comment]
Closing as `1.1.4` is out, if someone has problems with that please open a new issue :)
